.github/ISSUE_TEMPLATE/1_bug_report.yaml

```diff
@@ -29,13 +29,16 @@ body:
     id: versions
     attributes:
       label: What version are you seeing the problem on?
+      id: versions
+      description: select all version where you have experienced this issue
       multiple: true
       options:
-        - 1.6.x
-        - 1.7.x
-        - 1.8.x
-        - 1.9.x
-        - 2.0+
+        - "ver: 1.6.x"
+        - "ver: 1.7.x"
+        - "ver: 1.8.x"
```

Comment:
Is having 1.6, 1.7 and 1.8 useful at all? If we only support 1.9.x and 2.x moving forward, those should be the only options.

Then there could be a "Does this reproduce on master" checkbox.

Reply:
we can trace the root of the problem, and eventually after closing older issues...
having this information is not slowing us anyway...